docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -0,0 +1,320 @@
+    - `https://localhost` and `oci://localhost` URLs are now forbidden in `ApplicationDefinitions`. Since `localhost` would refer to the usercluster-controller-manager Pod, no such URLs should exist and the impact of this change should be non-existent
+- Add `AddonReconciledSuccessfully` condition / `Phase` to addons ([#13257](https://github.com/kubermatic/kubermatic/pull/13257))
+    - Add new `AddonReconciledSuccessfully` condition to Addon resources.
+    - Add `Phase` (New/Healthy/Unhealthy) to Addon resources (for informational purpose only, integrations should rely on the individual condition statusses) 
+- Bump Metering to 1.2.1 ([#13185](https://github.com/kubermatic/kubermatic/pull/13185))
+    - Add `format` to metering report configuration, allowing to generate JSON files instead of CSV.- Add `cloud-provider`, `datacenter` and `cluster-owner` columns to the generated metering reports
+- A new option to customize non-essential fields in the preset ([#13672](https://github.com/kubermatic/kubermatic/pull/13672))
+- Add `AllowedOperatingSystems` option for the project. This can be used to limit the allowed operating systems for a project ([#13442](https://github.com/kubermatic/kubermatic/pull/13442))
+- Add `DisplayName` for applications, this is the name displayed on the UI ([#13331](https://github.com/kubermatic/kubermatic/pull/13331))
+- Add Canal CNI version v3.27.3 ([#13239](https://github.com/kubermatic/kubermatic/pull/13239))
+- Add new `kubermatic_cluster_owner` metric on seed clusters, with `cluster_name` and `user` labels ([#13194](https://github.com/kubermatic/kubermatic/pull/13194))
+- Add new admin option to enable/disable etcd Backups ([#13355](https://github.com/kubermatic/kubermatic/pull/13355))
+- Allow to specify extra annotations for the Dex ingress ([#13188](https://github.com/kubermatic/kubermatic/pull/13188))
+- Introduce annotation configuration for the dashboard in `KubermaticSettings`. A List of protected and hidden annotations can now be configured for the dashboard ([#13668](https://github.com/kubermatic/kubermatic/pull/13668))
+- Introduce Cilium 1.15.3 and mitigate CVE-2024-28860 and CVE-2024-28248 in 1.14.9 and 1.13.14 ([#13241](https://github.com/kubermatic/kubermatic/pull/13241))
+- KKP resources in the `kubermatic.k8c.io` API Group can be annotated with `policy.k8c.io/prevent-deletion` to make the kubermatic-webhook reject any delete attempt (even by cluster-admins). This is meant as a last resort mechanism to prevent accidental deletions by admins during maintenance on a KKP system ([#13284](https://github.com/kubermatic/kubermatic/pull/13284))
+- Monitoring: introduce `signout_redirect_url` field to configure the URL to redirect the user to after signing out from Grafana ([#13313](https://github.com/kubermatic/kubermatic/pull/13313))
+- Support for configuring API server service type for the user clusters ([#13562](https://github.com/kubermatic/kubermatic/pull/13562))
```

Comment:
`apiserver` makes more sense here than "API server" (in Kubernetes land, it's simply called "apiserver").